lib/krane/errors.rb

```diff
@@ -0,0 +1,34 @@
+# frozen_string_literal: true
+
+module Krane
+  class FatalDeploymentError < StandardError; end
+  class FatalKubeAPIError < FatalDeploymentError; end
+  class KubectlError < StandardError; end
+  class TaskConfigurationError < FatalDeploymentError; end
+
+  class InvalidTemplateError < FatalDeploymentError
+    attr_reader :content
+    attr_accessor :filename
+    def initialize(err, filename: nil, content: nil)
+      @filename = filename
+      @content = content
+      super(err)
+    end
+  end
+
+  class NamespaceNotFoundError < FatalDeploymentError
```

Comment:
Not introduced by this PR, but I think this is error class is dead code?

Reply:
Good catch, removed it 👍